azurelinuxagent/ga/exthandlers.py

```diff
@@ -474,12 +472,8 @@ def handle_ext_handler_download_error(self, ext_handler_i, e, code=-1):
         msg = ustr(e)
         ext_handler_i.set_handler_status(message=msg, code=code)
 
-        self.get_artifact_error_state.incr()
```

Comment:
All the operations that raise `ExtensionDownloadError` only happen on a per GoalState basis. It doesn't make sense to not report them everytime if it causes the GoalState to fail. Changing the logging to report always rather than every 5 mins

Reply:
Seems like get_artifact_error_state can be removed altogether. The only usage I see is in

```
            self.ext_handlers, etag = self.protocol.get_ext_handlers()
            self.get_artifact_error_state.reset()
```

and get_ext_handler doesn't download anything

Reply:
Verified that get_artifact_error_state is no longer needed. We already log the error as part of `try_update_gs` function. Removed